build.sbt

```diff
@@ -1,6 +1,11 @@
 import Build.*
 
-import scala.collection.Seq
+inThisBuild(
+  List(
+    organization := "com.lunatech",
```

Comment:
Btw `organization` is also set in `commonSettings`, so you can remove it there and just have it here.